docs/source/models/concept_models/vivarium_nutrition_optimization/concept_model.rst

```diff
@@ -113,26 +113,134 @@ To achieve the general modeling objective of determining the most cost effective
 
-2.1.1 Process in practice
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Simulate 1 scenario for each possible combination of modeled interventions either at 0% or 100% coverage.
+2.2 Emulator Design
+-------------------
 
-2. External to vivarium, pair the simulated results of these scenarios with an intervention costing function and solve for combination of scenarios that maximizes health impact under the following constraints/assumptions:
+The emulator is designed to take in the results from the simulation and costing 
+functions and output an optimized approach for each budget. Below is more details 
+about how the emulator works, its input parameters, and limitations. 
 
-  - Constraint: total cost of resulting combination of interventions cannot exceed some set budget
-  - Constraint: total coverage of any one intervention cannot exceed that intervention-specific saturation coverage level
-  - Assumption: all interventions have the same coverage propensities. See the `6.0 Limitations`_ section for a discussion of this assumption.  
-  - Pending constraint: avoid operationally infeasible coverage recommendations (for example: separate antenatal care intervention product based on whether child nutrition saturation coverage has already been maxxed out)
+2.2.1 General Emulator Strategy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Step 1 in this process is detailed in the :ref:`pregnancy<2021_concept_model_vivarium_nutrition_optimization_pregnancies>` and :ref:`child<2021_concept_model_vivarium_nutrition_optimization_children>` concept model documents. Step 2 in this process will be performed using the outputs from step 1 and `can be found here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/tree/data_prep/emulator>`_
+To find optimized results, we will: 
+
+#. Simulate 1 scenario for each possible combination of modeled interventions either at 0% or 100% coverage.
+#. Define cost functions for each intervention. 
+#. External to vivarium, pair the simulated results of these scenarios with the intervention costing function and solve for combination of scenarios that maximizes health impact. 
+
+Further details can be found in the below sections. 
+
+Step 1 in this process is detailed in the :ref:`pregnancy<2021_concept_model_vivarium_nutrition_optimization_pregnancies>` and :ref:`child<2021_concept_model_vivarium_nutrition_optimization_children>` concept model documents. 
+
+
+2.2.2 Input Parameters
+~~~~~~~~~~~~~~~~~~~~~~
+
+We expect that the emulator will have the following input parameters 
+that are configurable for each run: 
+
+.. list-table:: Input Parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Options to include 
+    - Notes
+  * - Results directory
+    - 
+    - This will include results for only one location. The emulator will be rerun each location.
+  * - Location ID 
+    - 
+    - Accepts exactly one location ID 
```

Comment:
How do you expect location ID will be used? To influence costs? We might be able to automatically read in the location name from the results directory instead of having to input a location ID directly.

Reply:
Ok, gotcha, that makes sense! Yeah, the results directories typically have the location name in the directory (like below), so we could potentially pull that out of the directory and use get_ids() to find the location ID, but this sort of fancy thing is definitely less of a priority than other functionality 

'/mnt/team/simulation_science/costeffectiveness/results/vivarium_gates_nutrition_optimization/model5.0/ethiopia/2023_08_10_13_13_46/'